<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <remap from="/plan/speed" to="/speed"/>
    <remap from="/plan/steering" to="/steering"/>

    <include file="$(find avc)/launch/mapper.launch"/>
    <include file="$(find avc)/launch/planner_macaroni.launch"/>

    <!-- <node name="navigation_controller" pkg="rr_common" type="navigation_controller" output="screen">
        <param name="req_finish_line_crosses" type="int" value="2"/>
        <param name="startSignal" type="string" value="/start_detected"/>
        <param name="resetSignal" type="string" value="/reset_detected"/>
    </node> -->
</launch>
